src/guiengine/skin.cpp

```diff
@@ -1276,6 +1270,10 @@ void Skin::drawSpinnerBody(const core::recti &rect, Widget* widget,
     {
         params=&SkinConfig::m_render_params["spinner::neutral"];
     }
+    widget->m_skin_r = color_rgb.r * 255.0F;
+    widget->m_skin_g = color_rgb.g * 255.0F;
+    widget->m_skin_b = color_rgb.b * 255.0F;
```

Comment:
I never really checked if uppercase "F" will work, but it just looks strange. We use lowercase "f" everywhere.